README.md

```diff
@@ -24,12 +24,18 @@ npm install mime-db
 ### Database Download
 
 If you're crazy enough to use this in the browser, you can just grab the
-JSON file using [RawGit](https://rawgit.com/). It is recommended to replace
-`master` with [a release tag](https://github.com/jshttp/mime-db/tags) as the
-JSON format may change in the future.
+JSON file using [unpkg](https://unpkg.com/mime-db/db.json).
 
 ```
-https://cdn.rawgit.com/jshttp/mime-db/master/db.json
+# use the latest database 
+https://unpkg.com/mime-db/db.json
+```
+
+It is recommended to keep the version in the URL as the JSON format may change in the future.
+
+```
+# always use the v1.37.0 version
```

Comment:
We use to have an example with a specific version, but people copy it as is and wanted us to keep updating the version in the readme with every release. Instead it was a generic instruction. Can you make it like that with unpkg like it was with rawgit?

Reply:
Sure. How about the latest updates?

Reply:
Is it not possible to link to the list of versions like it used to? Also not as big of a deal, but it would be nice not to even have the 1 part, as it is too easy to publish a new version and forget this is there.

Reply:
Sure. I found that `jsDelivr` provides release tags in the URL like RawGit does so I've updated the links to `jsDelivr` now, the instructions are very similar to the RawGit one now.